### PR TITLE
Migrate gRPC source code to separate repo with history

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
Migrate gRPC source code to separate branch with history.

### CODE REVIEW

**Review by aashikam — APPROVED** [review]
LGTM

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        maven {
+            url = 'https://maven.wso2.org/nexus/content/repositories/orgballerinalang-1614'
+        }
+
+        maven {
+            url = 'https://repo.maven.apache.org/maven2'
+        }
+
+        maven {
+            url = 'https://maven.pkg.github.com/ballerina-platform/module-ballerinax-nats'
```
Comment (BuddhiWathsala) [inline build.gradle:73]:
```suggestion
            url = 'https://maven.pkg.github.com/ballerina-platform/module-ballerina-grpc'
```

File: grpc-ballerina/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,153 @@
+[project]
+org-name = "ballerina"
+version = "@toml.version@"
+
+[platform]
+target = "java8"
+
+    [[platform.libraries]]
+    artifactId = "grpc"
+    version = "0.7.0-SNAPSHOT"
```
Comment (BuddhiWathsala) [inline grpc-ballerina/Ballerina.toml:10]:
```suggestion
    version = "@project.version@"
```

File: grpc-ballerina/Ballerina.toml [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    path = "../grpc-native/build/libs/grpc-native-@project.version@.jar"
+    groupId = "ballerina"
+    modules = ["grpc"]
+
+    [[platform.libraries]]
+    artifactId = "grpc"
+    version = "0.7.0-SNAPSHOT"
```
Comment (BuddhiWathsala) [inline grpc-ballerina/Ballerina.toml:17]:
```suggestion
    version = "@project.version@"
```

File: grpc-test-utils/build.gradle [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ * limitations under the License.
+ *
+ */
+
+plugins {
+    id 'java'
+}
+
+group 'org.ballerinalang'
+version '0.7.0-SNAPSHOT'
```
Comment (BuddhiWathsala) [inline grpc-test-utils/build.gradle:23]: I guess we don't need this version and group ID here

**Review by BuddhiWathsala — CHANGES_REQUESTED** [review]

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**